src/main/java/com/bankxpvalue/BankXpValuePlugin.java

```diff
@@ -166,7 +139,7 @@ public void onConfigChanged(ConfigChanged configChanged){
             overlayManager.remove(itemOverlay);
```

Comment:
@dseeler what is this code for? Is this for `runelite.bankxpvalueplugin` aka when you toggle the plugin on or off? Shouldn't that be handled by `startUp` and `shutDown`?

Reply:
We need to keep this. `startUp` and `shutDown` are handlers for when the plugin _is_ enabled but should/should not be running. The code above is a higher-level event handler for when the plugin itself is turned off (`shutDown` does not handle this). It is necessary to revert the plugin back to its default state.

Reply:
In my testing `shutDown` is called when you turn the plugin off. I just put a log in `shutDown` and when i flip the toggle it prints it.